extended/src/main/java/apoc/ml/bedrock/ModelItemResult.java

```diff
@@ -0,0 +1,31 @@
+package apoc.ml.bedrock;
+
+import java.util.List;
+import java.util.Map;
+
+public class ModelItemResult {
```

Comment:
questo perche non record?

Reply:
Perché essendoci parecchi parametri, mettendo record dovrei mettere un costruttore tutti i parametri, tipo:
```
public static ModelItemResult from(Map<String, Object> map) {
String modelId = (String) map.get("modelId");
// simile per gli altri

// costruttore con tutti i parametri
        return new ModelItemResult(modelId,
                modelArn,
                modelName,
                providerName,
                responseStreamingSupported,
                customizationsSupported,
                inferenceTypesSupported,
                inputModalities,
                outputModalities);
```

Con class mi sembra più pulito, facendo semplicemente `this.modelName = (String) map.get("modelName");`, nel costruttore